### ISSUE TITLE
Crash static version on centos 7

### ISSUE THREAD

**Issue by alpody** [xo/usql#262]
I use Centos 7

[webmaster@localhost bin]$ uname -a
Linux localhost.localdomain 3.10.0-1127.19.1.el7.x86_64 #1 SMP Tue Aug 25 17:23:54 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

When I run static version of usql (0.9.1), I have SIGABRT

[webmaster@localhost bin]$ usql_static
usql: dl-call-libc-early-init.c:37: _dl_call_libc_early_init: Assertion `sym != NULL' failed.
SIGABRT: abort
PC=0x1da2e3b m=0 sigcode=18446744073709551610
signal arrived during cgo execution

goroutine 1 [syscall, locked to thread]:
runtime.cgocall(0x1bac600, 0xc0000c5450, 0xc0000100f0)
        /usr/local/go/src/runtime/cgocall.go:154 +0x5b fp=0xc0000c5420 sp=0xc0000c53e8 pc=0x40b1bb
os/user._Cfunc_mygetpwuid_r(0x3e8, 0xc000097f50, 0x66fc780, 0x400, 0xc0000100f0, 0x7f2c00000000)
        _cgo_gotypes.go:172 +0x48 fp=0xc0000c5450 sp=0xc0000c5420 pc=0x6057a8
os/user.lookupUnixUid.func1.1(0x3e8, 0xc000097f50, 0xc0001665f0, 0xc0000100f0, 0xc0000c54f0)
        /usr/local/go/src/os/user/cgo_lookup_unix.go:103 +0xd0 fp=0xc0000c54a0 sp=0xc0000c5450 pc=0x606cf0
os/user.lookupUnixUid.func1(0x209cb60)
        /usr/local/go/src/os/user/cgo_lookup_unix.go:103 +0x45 fp=0xc0000c54d8 sp=0xc0000c54a0 pc=0x606d65
os/user.retryWithBuffer(0xc0001665f0, 0xc0000c55b0, 0x0, 0x2)
        /usr/local/go/src/os/user/cgo_lookup_unix.go:247 +0x3e fp=0xc0000c5530 sp=0xc0000c54d8 pc=0x6062be
os/user.lookupUnixUid(0x3e8, 0x0, 0x0, 0x0)
        /usr/local/go/src/os/user/cgo_lookup_unix.go:96 +0x132 fp=0xc0000c55f8 sp=0xc0000c5530 pc=0x605bd2
os/user.current(0xc00014a8f0, 0xc0000c5698, 0x4c01fe)
        /usr/local/go/src/os/user/cgo_lookup_unix.go:49 +0x49 fp=0xc0000c5638 sp=0xc0000c55f8 pc=0x605a69
os/user.Current.func1()
        /usr/local/go/src/os/user/lookup.go:15 +0x25 fp=0xc0000c5660 sp=0xc0000c5638 pc=0x606ba5
sync.(*Once).doSlow(0x55a9060, 0x23fc9e0)
        /usr/local/go/src/sync/once.go:68 +0xec fp=0xc0000c56b0 sp=0xc0000c5660 pc=0x49088c
sync.(*Once).Do(...)
        /usr/local/go/src/sync/once.go:59
os/user.Current(0x228686d, 0x8, 0x0)
        /usr/local/go/src/os/user/lookup.go:15 +0x105 fp=0xc0000c56e0 sp=0xc0000c56b0 pc=0x6054c5
github.com/xo/terminfo.Load(0xc000044005, 0xf, 0xc000044005, 0xf, 0xffffffffffffffff)
        /home/ken/src/go/pkg/mod/github.com/xo/terminfo@v0.0.0-20210125001918-ca9a967f8778/load.go:43 +0x119 fp=0xc0000c57b8 sp=0xc0000c56e0 pc=0x85ff59
github.com/xo/terminfo.ColorLevelFromEnv(0xc0000c58b8, 0x1, 0x1)
        /home/ken/src/go/pkg/mod/github.com/xo/terminfo@v0.0.0-20210125001918-ca9a967f8778/color.go:76 +0x205 fp=0xc0000c5838 sp=0xc0000c57b8 pc=0x85fba5
github.com/xo/usql/env.init.0()
        /home/ken/src/go/src/github.com/xo/usql/env/types.go:44 +0x91 fp=0xc0000c58f8 sp=0xc0000c5838 pc=0x866cf1
runtime.doInit(0x547af00)
        /usr/local/go/src/runtime/proc.go:6309 +0xec fp=0xc0000c5a48 sp=0xc0000c58f8 pc=0x44eb6c
runtime.doInit(0x546a2a0)
        /usr/local/go/src/runtime/proc.go:6286 +0x72 fp=0xc0000c5b98 sp=0xc0000c5a48 pc=0x44eaf2
runtime.doInit(0x54680a0)
        /usr/local/go/src/runtime/proc.go:6286 +0x72 fp=0xc0000c5ce8 sp=0xc0000c5b98 pc=0x44eaf2
runtime.doInit(0x5471d40)
        /usr/local/go/src/runtime/proc.go:6286 +0x72 fp=0xc0000c5e38 sp=0xc0000c5ce8 pc=0x44eaf2
runtime.doInit(0x546dfa0)
        /usr/local/go/src/runtime/proc.go:6286 +0x72 fp=0xc0000c5f88 sp=0xc0000c5e38 pc=0x44eaf2
runtime.main()
        /usr/local/go/src/runtime/proc.go:208 +0x205 fp=0xc0000c5fe0 sp=0xc0000c5f88 pc=0x440f05
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1371 +0x1 fp=0xc0000c5fe8 sp=0xc0000c5fe0 pc=0x4758c1

rax    0x0
rbx    0x66e93c0
rcx    0xffffffffffffffff
rdx    0x0
rdi    0x2
rsi    0x7ffe7a79a320
rbp    0x48107b0
rsp    0x7ffe7a79a320
r8     0x0
r9     0x7ffe7a79a320
r10    0x8
r11    0x246
r12    0x481a28e
r13    0x25
r14    0x481a2a8
r15    0x66fcea0
rip    0x1da2e3b
rflags 0x246
cs     0x33
fs     0x0
gs     0x0

**Comment by nineinchnick** [comment]
I can repro this by running:
```bash
% tar xf usql_static-0.9.1-linux-amd64.tar.bz2
% docker run --rm -it -v $(pwd):/app centos:7
[root@02f464a91d2c /]# /app/usql_static
```

It happens when fetching the current user and Go makes a syscall to `mygetpwuid`:
```
return syscall.Errno(C.mygetpwuid_r(C.int(uid),
			&pwd,
			(*C.char)(buf.ptr),
			C.size_t(buf.size),
			&result))
```

Looks like CentOS has a different version of glibc than we used during build and we don't have the `osusergo` build tag. It was removed in #251 but only netgo was added back after it in 521f8f54883d2f0410e96f62eea5c5e6d2eaf2ad. I think we can consider also adding back `osusergo`.

**Comment by nineinchnick** [comment]
This was recently fixed on master in https://github.com/xo/usql/commit/2e2ff5edc968d7c19230e7cce90891449e5cf05e, pending a new release.

**Comment by nineinchnick** [comment]
This should be fixed in [0.9.2](https://github.com/xo/usql/releases/tag/v0.9.2)
